payloads/external/Makefile.mk

```diff
@@ -168,7 +168,7 @@ IPXE_EFI := payloads/external/iPXE/ipxe/ipxe.rom
 endif
 
 $(obj)/UEFIPAYLOAD.fd: $(DOTCONFIG) $(IPXE_EFI)
-	$(MAKE) -C payloads/external/edk2 UefiPayloadPkg \
+	$(MAKE) -C payloads/external/edk2 DasharoPayloadPkg \
```

Comment:
This commit (or an identical one) was actually done for CI in https://github.com/Dasharo/coreboot/pull/486 as I assumed we'll just switch to `DasharoPayloadPkg` everywhere in one go.  If we want to have QEMU board to be the only user of rebased EDK for now, then need some new `Kconfig` option to set build target name and override `PAYLOAD_NAME` in `edk2/Makefile`.

Reply:
Dropped and changed PR base.